css/icons.scss

```diff
@@ -53,6 +53,10 @@
   background-image: url('../img/toggle-view-collapse.svg');
 }
 
+.icon-clone {
+  background-image: url('../img/clone.svg');
```

Comment:
Please use the icon-black-white mixin similar to line 61:

```
@include icon-black-white('deck', 'clone', 1);
```

Reply:
thanks, it's working :)